share/wake/lib/core/string.wake

```diff
@@ -56,6 +56,15 @@ export def explode (string: String): List String =
     def p s = prim "explode"
     p string
 
+# truncate: return only the first N Unicode code points from a given string.
+#
+#   truncate 7 "hello world" = "hello w"
```

Comment:
Needs more formatting, see list.wake for blessed examples. I realize you've matched the surrounding functions but I'd prefer to move things to the list.wake style now. Of particular importance is surrounding examples with triple back-ticks so that editors format it all nicely.

I'd also show what happens for edge cases like -10, the integer being over the string size, and 0